skore/tests/unit/sklearn/train_test_split/test_train_test_split.py

```diff
@@ -180,3 +180,41 @@ def test_train_test_split_kwargs():
     output2 = train_test_split(X=X, y=y, random_state=0)
 
     assert output1 == output2
+
+
+def test_train_test_split_dict_kwargs():
+    """Passing data without keyword arguments with return_dict=True
+    should raise ValueError."""
+
+    X = [[1]] * 20
+    y = [0] * 10 + [1] * 10
+
+    with pytest.raises(ValueError):
```

Comment:
Just to be sure that the right ValueError is raised:
```suggestion
    with pytest.raises(ValueError, match="When return_dict=True, arrays must be passed as keyword arguments"):
```

Reply:
Added in https://github.com/probabl-ai/skore/pull/1460/commits/9b1a875b869f49ed0bcccdd80777c19502345226